src/EnergyPlus/SingleDuct.cc

```diff
@@ -3066,13 +3078,28 @@ void SingleDuctAirTerminal::SizeSys(EnergyPlusData &state)
                 MinAirFlowFracDes = 0.0;
             }
         }
+        if (state.dataSize->SysSizingRunDone) {
+            if (state.dataSize->SysSizInput(SysSizNum).SystemOAMethod == SOAM_SP) { // 62.1 simplified procedure
+                if (this->MaxAirVolFlowRate > 0.0) {
+                    MinAirFlowFracDes = 1.5 *
+                                        max(state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).VozClgByZone,
+                                            state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).VozHtgByZone) /
+                                        this->MaxAirVolFlowRate;
+                }
+            }
+        }
```

Comment:
Calculation of the zone minimum primary air flow fraction following the Simplified Procedure using the maximum cooling or heating zone outdoor air flow.